src/python/pants/backend/go/go_sources/BUILD

```diff
@@ -0,0 +1,4 @@
+# Copyright 2021 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+resources(sources=["*.go", "!gentestflags.go"])
```

Comment:
Not for this PR, but could we support `go_package` here so we can add unit tests for this Go code? One blocker is that Pants developers might need to have Go installed.

(And the code would then need to eventually be organized in separate directories for separate binaries.)

Reply:
Yes for sure! We also would need a `go.mod`. Is that fine?

Reply:
That should be fine. I would need to reorganize the sources to retain the one `go_package` per directory rule and what not. I'll do that next week when I introduce some tests.